packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/blob/BlobProvider.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.react.modules.blob
+
+import android.content.ContentProvider
+import android.content.ContentValues
+import android.database.Cursor
+import android.net.Uri
+import android.os.ParcelFileDescriptor
+import com.facebook.infer.annotation.Nullsafe
+import com.facebook.react.ReactApplication
+import java.io.FileNotFoundException
+import java.io.IOException
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+
+public final class BlobProvider : ContentProvider() {
+  private val executor: ExecutorService = Executors.newSingleThreadExecutor()
+
+  override fun onCreate(): Boolean {
+    return true
+  }
+
+  override fun query(
+    uri: Uri,
+    projection: Array<String>?,
```

Comment:
This wasn't nullable in Java, can you double check this?

```suggestion
    projection: Array<String>,
```

Same for `selectionArgs `

Reply:
i have kept projection nullable (Array<String>?) to match the method signature of ContentProvider.query() in Kotlin. and similarly for others